docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -337,6 +337,17 @@ this table:
 
 No other types are supported.
 
+.. _delta-lake-table-redirection:
+
+Table redirection
+-----------------
+
+.. include:: table-redirection.fragment
+
+The connector supports redirecting the query to :doc:`/connector/hive`
+through the configuration setting ``delta.hive-catalog-name``
+when dealing with a non Delta Lake table.
```

Comment:
Phrasing nit, but I defer to the docs team. I find it clearer to put the situation you're addressing at the beginning of the sentence rather than at the end. 

```suggestion
When dealing with a non Delta Lake table, the connector supports redirecting queries to :doc:`/connector/hive`
using the configuration setting ``delta.hive-catalog-name``.
```